consensus/polybft/stale_sequence_check.go

```diff
@@ -0,0 +1,89 @@
+package polybft
+
+import (
+	"sync"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+)
+
+type staleSequenceCheck struct {
```

Comment:
Is this staying here for now?

Reply:
For now, until we make a better solution through another PR.